build.py

```diff
@@ -1100,21 +1105,21 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
             cuda_arch = 'sbsa' if target_machine == 'aarch64' else 'x86_64'
             df += '''
 RUN mkdir -p /usr/local/cuda/lib64/stubs
-COPY --from=min_container /usr/local/cuda/lib64/stubs/libcusparse.so /usr/local/cuda/lib64/stubs/libcusparse.so.11
+COPY --from=min_container /usr/local/cuda/lib64/stubs/libcusparse.so /usr/local/cuda/lib64/stubs/libcusparse.so.12
 COPY --from=min_container /usr/local/cuda/lib64/stubs/libcusolver.so /usr/local/cuda/lib64/stubs/libcusolver.so.11
```

Comment:
Why this library and the library below it has not been updated?

Reply:
It's a third party dependency and in CUDA container it actually present as without version suffix. 
And my understanding is that needed due to a compilation process of TRT which is keep using version in suffix. 
So my check shows the following.
```
$ docker run --rm -it gitlab-master.nvidia.com:5005/dl/dgx/pytorch:23.01-py3-base  bash
root@91209ca429c5:/workspace# ldd /usr/local/lib/python3.8/dist-packages/torch*/lib/* | grep -E ":|libcuso|libcurand"
/usr/local/lib/python3.8/dist-packages/torch/lib/libc10.so:
/usr/local/lib/python3.8/dist-packages/torch/lib/libc10_cuda.so:
/usr/local/lib/python3.8/dist-packages/torch/lib/libcaffe2_nvrtc.so:
/usr/local/lib/python3.8/dist-packages/torch/lib/libshm.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f835f6a1000)
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007fa488b94000)
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch_cpu.so:
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch_cuda.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f0efcb38000)
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch_cuda_linalg.so:
	libcusolver.so.11 => /usr/local/cuda/lib64/libcusolver.so.11 (0x00007f346884e000)
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f3443124000)
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch_global_deps.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f3586c2e000)
/usr/local/lib/python3.8/dist-packages/torch/lib/libtorch_python.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f497c014000)
/usr/local/lib/python3.8/dist-packages/torch_tensorrt/lib/libtorchtrt.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f02e9990000)
/usr/local/lib/python3.8/dist-packages/torch_tensorrt/lib/libtorchtrt_plugins.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007f44cf100000)
/usr/local/lib/python3.8/dist-packages/torch_tensorrt/lib/libtorchtrt_runtime.so:
	libcurand.so.10 => /usr/local/cuda/lib64/libcurand.so.10 (0x00007fabe4723000)
```